lib/node_modules/@stdlib/math/base/ops/imuldw/docs/types/test.ts

```diff
@@ -44,18 +43,88 @@ import imuldw = require( './index' );
 	imuldw( ( x: number ): number => x, 0x55555555 ); // $ExpectError
 }
 
-// The compiler throws an error if the function is provided an output array which is not array-like...
-{
-	imuldw( true, 0xAAAAAAAA, 0x55555555 ); // $ExpectError
-	imuldw( false, 0xAAAAAAAA, 0x55555555 ); // $ExpectError
-	imuldw( 'abc', 0xAAAAAAAA, 0x55555555 ); // $ExpectError
-	imuldw( {}, 0xAAAAAAAA, 0x55555555 ); // $ExpectError
-	imuldw( ( x: number ): number => x, 0xAAAAAAAA, 0x55555555 ); // $ExpectError
-	imuldw( 123, 0xAAAAAAAA, 0x55555555 ); // $ExpectError
-}
-
 // The compiler throws an error if the function is provided an insufficient number of arguments...
 {
 	imuldw(); // $ExpectError
 	imuldw( 1 ); // $ExpectError
 }
+
+// Attached to the main export is an `assign` method which returns an array-like object containing numbers...
+{
+	const out = [ 0.0, 0.0 ];
```

Comment:
This is being pedantic, but, because this API only returns integers, can we update L54 and below to `[ 0, 0 ]`? This would make this consistent with conventions in documentation elsewhere.